lib/ain-grpc/src/rpc/eth.rs

```diff
@@ -1133,6 +1141,41 @@ impl MetachainRPCServer for MetachainRPCModule {
     fn new_pending_transaction_filter(&self) -> RpcResult<U256> {
         Ok(self.handler.filters.create_tx_filter().into())
     }
+
+    fn create_access_list(
+        &self,
+        call: CallRequest,
+        block_number: Option<BlockNumber>,
+    ) -> RpcResult<AccessListResult> {
+        let caller = call.from.unwrap_or_default();
+        let byte_data = call.get_data()?;
+        let data = byte_data.0.as_slice();
+
+        // Get gas
+        let block_gas_limit = ain_cpp_imports::get_attribute_values(None).block_gas_limit;
+        let gas_limit = u64::try_from(call.gas.unwrap_or(U256::from(block_gas_limit)))
+            .map_err(to_custom_err)?;
+
+        let block = self.get_block(block_number)?;
+        let block_base_fee = block.header.base_fee;
+        let gas_price = call.get_effective_gas_price(block_base_fee)?;
+
+        Ok(self
+            .handler
+            .core
+            .create_access_list(EthCallArgs {
+                caller,
+                to: call.to,
```

Comment:
The to field in eth_createAccessList RPC is a required field when passing the call context for this RPC. We should check that the field is populated and if it isnt, return the appropriate RPC error